### ISSUE TITLE
3.6.3 Unable to export products

### ISSUE THREAD

**Issue by Bobspadger** [Flagbit/Magento-FACTFinder#114]
We are currently attempting to upgrade to the 3.6.3 release to resolve the fatal errors.

Whilst the cron export works, and the price / stock exports work as well, the main product feed is failing.

We just get the following:
```
Another export is already running. Please wait for it to finish before you start a new export.
If you receive this message after another export has failed, please delete the file "ffexport.lock" within your Magento directory.
```

Another export is definitely not running , as us triggering the export creates the FF lock file.
We have increased timeouts / memory limits etc but just get no data sent, then finally after some time the lock message.

**Comment by xpoback** [comment]
@Bobspadger Did I understand you correctly, at the beginning there's no lock file, then you run the export,  the lock file gets created and after some time you get this message in the browser?

**Comment by Bobspadger** [comment]
@xpoback 

Process = 
Trigger Export
ff_lock is created
Browser / curl sits and does nothing for ages (looking like its trying to process)
at point X in the future, we get the 'there is an export running already'

**Comment by xpoback** [comment]
@Bobspadger How long is for ages in this particular case?

**Comment by Bobspadger** [comment]
@xpoback 5minutes approximately

Is there an easy way to find the current version on our production site ? (sorry, can't see a note in our git commit)

**Comment by Bobspadger** [comment]
If I look in app/code/community/Flagbit/FactFinder/etc

I get a version of 3.5.5

**Comment by xpoback** [comment]
Unfortunately, the version in config.xml is not relevent at the moment in the 3.* extension.
The only way is to check the git tag.

The issue you discribe looks like some changes have slowed down and\or broken the export in your particular case.
How many websites do you have?

**Comment by Bobspadger** [comment]
@xpoback We only have 1 website.

I don't have any tags in the repo for this. :(

**Comment by bobbyshaw** [comment]
@xpoback The git tag that's deployed to @Bobspadger's site is 3.6.3 (same as issue name and initial description).

**Comment by Bobspadger** [comment]
@xpoback judging by the release dates of the modules, 3.5.6 was released in November 2014.
We upgraded to the current latest release in June/July 2015 so would stand to reason it was this one.

**Comment by xpoback** [comment]
@Bobspadger Sorry for the delay.
The issue is not easy to debug since we can't reproduce it.

Can you please use the solution from this commit https://github.com/Flagbit/Magento-FACTFinder/commit/30d0699c21be1ad325d49193e8e1cfc0b1ea9904 to fix the fatal errors you had in #112  instead of updating the whole module? As far as I guess that was the only reason for the update.

**Comment by xpoback** [comment]
@Bobspadger What's the status of this?
